opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -93,6 +94,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
     { name: 'description', type: 'string', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false },
     { name: 'firstname', type: 'string', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false },
     { name: 'lastname', type: 'string', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false },
+    { name: 'user_confidence_level', type: 'object', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, schemaDef: userConfidenceSchema },
```

Comment:
Just out of curiosity, why did you choose to put the user_confidence_level attribute in internalObjectsAttributes instead of the User attributes directly ?

Reply:
My bad, in my memory User was register in a graphql module and not like the old entities. So you are definetly in the good file ! :D